packages/core/src/App/Components/Layout/Header/toggle-menu-drawer.jsx

```diff
@@ -337,9 +337,7 @@ const ToggleMenuDrawer = React.forwardRef(
                                                     toggleTheme(!is_dark_mode);
                                                 }}
                                             >
-                                                <div
-                                                    className={classNames('header__menu-mobile-link')}
-                                                >
+                                                <div className={classNames('header__menu-mobile-link')}>
```

Comment:
What is this?

Reply:
The line got formatted when I ran `npm run prettify`